src/server/game/Entities/Creature/Creature.cpp

```diff
@@ -2930,7 +2930,7 @@ void Creature::UpdateMovementFlags()
         return;
 
     // Set the movement flags if the creature is in that mode. (Only fly if actually in air, only swim if in water, etc)
-    float ground = GetFloorZ();
+	float ground = GetMap()->GetHeight(GetPhaseMask(), GetPositionX(), GetPositionY(), GetPositionZMinusOffset());
```

Comment:
Ok I see, VS2017 and Cntrl + K + D = Autoformat with resharper which caused the tab